apps/05Mastermind.js

```diff
@@ -53,17 +55,22 @@ function generateHint(solution, guess) {
 
 function mastermind(guess) {
   // your code here
```

Comment:
Hi Judah! I am not getting all of the tests to pass on my side. Were you able to pass them on yours?

abcd
[ null, null, 'c', 'd' ]
    ✓ should register a guess and generate hints
abcd
[ null, null, null, null ]
4-0
    1) should be able to detect a win

  #generateHint()
[ null, null, null, null ]
    2) should generate hints
[ null, null, 'c', 'd' ]
    3) should generate hints if solution has duplicates

Also no need to add toString(). when you do correctLetterAndLocation+"-", correctLetterAndLocation is automatically turned into a string.